h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/BoosterParms.java

```diff
@@ -43,6 +43,7 @@ public static BoosterParms fromMap(Map<String, Object> map) {
   private static Map<String, Object> localizeDecimalParams(final Map<String, Object> params) {
     Map<String, Object> localized = new HashMap<>(params.size());
     final NumberFormat localizedNumberFormatter = DecimalFormat.getNumberInstance(Locale.ENGLISH);
```

Comment:
should we really use localized number format instead of some IEEE standard number format?

Reply:
Which IEEE standard do you have in mind ? The point here is to make sure decimal point is unified.

Reply:
hmm, good question. then should we just use new DecimalFormat("0.0;-0.0") if English is really not what we want?

Reply:
That would have the same effect indeed.

Reply:
Won't work. Try it out. It will still use the default locale.

Reply:
Why is Double.toString() not enough? :)

Reply:
It's the same. I'll do it for you.

Reply:
This is already resolved.